web-ui/modules/i18n/locales/ja.mjs

```
@@ -0,0 +1,1081 @@
+    'common.refresh': '更新',
+    'common.refreshing': '更新中...',
+    'common.loading': '読み込み中...',
+    'common.saving': '保存中...',
+    'common.sending': '送信中...',
+    'common.scanning': 'スキャン中...',
+    'common.export': 'エクスポート',
+    'common.import': 'インポート',
+    'common.apply': '適用',
+    'common.applying': '適用中...',
+    'common.confirming': '確認中...',
+    'common.writeToEditor': 'エディタに書き込み',
```

Biome (2.4.15)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)